documentation/guides/getting-started.md

```diff
@@ -47,6 +48,6 @@ Visually, all Spectrum Web Components are an expression of the design tokens tha
 
 ## What you can do
 
-From this baseline, visit the documentation for each individual component package and find one, some, or all of those that are right for your project. Start building today and use rapid, component-based prototyping to bring your designs to life faster than ever.
+Now that you have a starting point, visit the documentation for each package and find some components that are right for your project. You can take a look at our examples on webcomponents.dev to get an idea of how these components behave in a development environment, or browse through our <sp-link href="https://opensource.adobe.com/spectrum-web-components/storybook/" target="_blank">storybook</sp-link>. Once you start developing, you’ll find that rapid, component-based prototyping brings your designs to life faster than ever.
```

Comment:
Can you link `webcomponents.dev` to https://webcomponents.dev/workspace/adobe?collection=spectrum-web-components

Reply:
Whoops, forgot about this. Added